### PR TITLE
Issue #2008: Rephrase stop time message

### PR DESCRIPTION
**Pull request by emmanuelbernard**

### CODE REVIEW

File: core/runtime/src/main/java/io/quarkus/runtime/Timing.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public static void printStopTime() {
         final long stopTimeNanoSeconds = System.nanoTime() - bootStopTime;
         final Logger logger = Logger.getLogger("io.quarkus");
         final BigDecimal secondsRepresentation = convertToBigDecimalSeconds(stopTimeNanoSeconds);
-        logger.infof("Quarkus stopped in %ss", secondsRepresentation);
+        logger.infof("Quarkus ran for %ss", secondsRepresentation);
```
Comment (gsmet) [inline core/runtime/src/main/java/io/quarkus/runtime/Timing.java]: Hmmm, seeing `final long stopTimeNanoSeconds = System.nanoTime() - bootStopTime;` I think there's something else going on.

**Review by gsmet — CHANGES_REQUESTED** [review]
This doesn't seem right. The code seems correct and seems to compute the stopping time.

And when I do:
```
$ java -jar /data/home/gsmet/git/quarkus-quickstarts/rest-json/target/rest-json-1.0-SNAPSHOT-runner.jar
```
Then wait 10 seconds then stop, I get:
```
^C2019-04-11 19:23:09,575 INFO  [io.quarkus] (main) Quarkus stopped in 0.009s
```

So exactly what I would expect.

How can I reproduce it? Is the issue with native?

**Comment by emmanuelbernard** [comment]
I encountered the long numbers in ˋquarkus:dev` mode after a few rounds of redeploy.

**Comment by stuartwdouglas** [comment]
I think https://github.com/quarkusio/quarkus/pull/2018 is the fix. The methods that set the time have a guard so it can only be set once.

**Pull request closed without merging**